kubeadm/scripts/PrepareNode.ps1

```diff
@@ -51,9 +57,13 @@ DownloadFile $kubeletBinPath https://dl.k8s.io/$KubernetesVersion/bin/windows/am
 DownloadFile "$global:KubernetesPath\kubeadm.exe" https://dl.k8s.io/$KubernetesVersion/bin/windows/amd64/kubeadm.exe
 DownloadFile "$global:KubernetesPath\wins.exe" https://github.com/rancher/wins/releases/download/v0.0.4/wins.exe
 
-# Create host network to allow kubelet to schedule hostNetwork pods
-Write-Host "Creating Docker host network"
-docker network create -d nat host
+if ($ContainerRuntime -eq "Docker") {
```

Comment:
There should probably be a check that Containerd/docker is installed on the machine.  Could this script call Install-containerd?

Reply:
Since Install-Containerd.ps1 may need to be run mutliple times I'd rather not call it.
We can check if the service is installed tho.

Reply:
Why would Install-Containerd.ps1 need to be called more than once?  

Checking for the service makes sense.

Reply:
If the required windows features are not enabled the script enables them and then prompts you to reboot and re-run the script.